Sequence.java

```
@@ -0,0 +1,34 @@
+import java.util.*;
+
+public class Sequence {
+  public static void main (String[] args) {
+     System.out.println("Enter number with a comma");
+     Scanner scan = new Scanner(System.in);
+     String input = scan.nextLine();
+     String[] number = input.split(",");
+     int[] sequence = new int[number.length];
+     for (int i = 0; i < number.length; i++) {
+       try {
+                sequence[i] = Integer.parseInt(number[i]);
+            } catch (NumberFormatException e) {
+                continue;

```

never ignore exception - at least you must log that something went wrong